src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs

```diff
@@ -133,6 +133,11 @@ protected HttpClient CreateHttpClientForRemoteServer(Configuration.Http.RemoteSe
 
             var client = new HttpClient(wrappedHandler);
             SetDefaultRequestVersion(client, remoteServer.HttpVersion);
+            if (timeout.HasValue)
```

Comment:
Rather than pass this in and set it here, can we just have callers set it directly on the HttpClient? This is generally what we do for other settings on HttpClient.

Reply:
Good point, that should simplify the changes significantly.